### ISSUE TITLE
missing return

### ISSUE THREAD

**Issue by bvella** [Coreoz/parcel-transformer-ttypescript#1]
- **I'm submitting a ...**
  [*] bug report
  [ ] feature request
  [ ] question about the decisions made in the repository
  [ ] question about how to use this project

- **Summary**
I tried using this and had to add a return to line 7 of the script for it to work. 

```
async loadConfig({config, options}) {
    return await loadTSConfig(config, options);
},
```

Could you apply the fix and push out a new version? Thanks

- **Other information** (e.g. detailed explanation, stack traces, related issues, suggestions how to fix, links for us to have context, eg. StackOverflow, personal fork, etc.)

**Comment by amanteaux** [comment]
Hello @bvella,

Can you make a PR? I'll merge it a make a release.

Though for information, this project was really a POC to use Parcel with ttypescript. But in the end, Parcel has been a failure for us, it contained many bugs and it was super slow.
We are now using:
- raw `ttsc` to compile the TS sources
- `vitejs` for the bundler (to bundle sources compiled from `ttsc`); `vite` is by far superior to parcel...
You may find an example of this setup by creating a project using this cli https://github.com/Coreoz/create-plume-react-project

So about parcel-transformer-ttypescript:
- It is a dead project for us
- If you want to maintain it, feel free to create a clone of this project:
  - I'll reference in the `README.md` your clone
  - I can grant you the access to publish new releases on npm


Cheers,
Aurélien

**Comment by amanteaux** [comment]
A 1.0.2 version has been released with #2 to fix this issue: https://www.npmjs.com/package/parcel-transformer-ttypescript

But it is important to note that this library is not maintained anymore by Coreoz: Parcel does not meet our quality requirements. As an example, the latest version of Parcel, the 2.2.0, breaks all the typescript types... I had to rollback to the version 2.0.0-beta1 to be able to compile the library.

Vitejs has been a great replacement of Parcel for us: https://github.com/vitejs/vite
